internal/daemonset/daemonset.go

```diff
@@ -340,42 +364,69 @@ func OverrideLabels(labels, overrides map[string]string) map[string]string {
 }
 
 func MakeLoadCommand(spec kmmv1beta1.ModprobeSpec) []string {
-	loadCommand := []string{"modprobe"}
+	loadCommandShell := []string{
+		"/bin/sh",
+		"-c",
+	}
+
+	loadCommand := "modprobe"
 
 	if ra := spec.RawArgs; ra != nil && len(ra.Load) > 0 {
-		return append(loadCommand, ra.Load...)
+		loadCommand = fmt.Sprintf("%s %s", loadCommand, strings.Join(ra.Load[:], " "))
+		return append(loadCommandShell, loadCommand)
+	}
+
+	if fw := spec.FirmwarePath; fw != "" {
+		loadCommand = fmt.Sprintf("cp -r %s %s && %s", fw, nodeVarLibFirmwarePath, loadCommand)
 	}
 
 	if a := spec.Args; a != nil && len(a.Load) > 0 {
-		loadCommand = append(loadCommand, a.Load...)
+		loadCommand = fmt.Sprintf("%s %s", loadCommand, strings.Join(a.Load[:], " "))
```

Comment:
I think using a `strings.Builder` would more efficient in `MakeLoadCommand` and `MakeUnloadCommand`.

Reply:
Not sure what it would look like. Could you provide an example, please?

Reply:
Opened #44 to follow-up on this.